src/app/hooks/useDetectOrdinalInSignPsbt.ts

```diff
@@ -1,33 +1,41 @@
 import {
   Bundle,
-  BundleSatRange,
   getUtxoOrdinalBundle,
   mapRareSatsAPIResponseToBundle,
   ParsedPSBT,
 } from '@secretkeylabs/xverse-core';
 import useWalletSelector from './useWalletSelector';
 
-export type InputsBundle = Pick<Bundle, 'value' | 'satRanges' | 'totalExoticSats'>;
+export type InputsBundle = (Pick<Bundle, 'satRanges' | 'totalExoticSats'> & {
+  inputIndex: number;
+})[];
 
 const useDetectOrdinalInSignPsbt = () => {
   const { ordinalsAddress, network } = useWalletSelector();
 
-  const handleOrdinalAndOrdinalInfo = async (parsedPsbt?: ParsedPSBT) => {
-    const satRanges: BundleSatRange[] = [];
-    let value = 0;
-    let totalExoticSats = 0;
+  const handleOrdinalAndOrdinalInfo = async (
+    parsedPsbt?: ParsedPSBT,
+  ): Promise<{ bundleItemsData: InputsBundle; userReceivesOrdinal: boolean }> => {
+    const bundleItemsData: InputsBundle = [];
     let userReceivesOrdinal = false;
 
     if (parsedPsbt) {
       const inputsRequest = parsedPsbt.inputs.map((input) =>
         getUtxoOrdinalBundle(network.type, input.txid, input.index),
       );
       const inputsResponse = await Promise.all(inputsRequest);
-      inputsResponse.forEach((inputResponse) => {
+      inputsResponse.forEach((inputResponse, index) => {
         const bundle = mapRareSatsAPIResponseToBundle(inputResponse);
-        value += bundle.value;
-        totalExoticSats += bundle.totalExoticSats;
-        satRanges.push(...bundle.satRanges);
+        if (
+          bundle.inscriptions.length > 0 ||
+          bundle.satributes.some((satributes) => !satributes.includes('COMMON'))
```

Comment:
what if `bundle.satributes: ['COMMON', 'UNCOMMON']`

it won't get pushed into bundleItemsData?